posthog/models/event/sql.py

```diff
@@ -111,6 +112,7 @@ def DROP_EVENTS_TABLE_SQL():
     , $group_4 VARCHAR COMMENT 'column_materializer::$group_4'
     , $window_id VARCHAR COMMENT 'column_materializer::$window_id'
     , $session_id VARCHAR COMMENT 'column_materializer::$session_id'
+    , $session_id_uuid Nullable(UInt128) COMMENT 'column_materializer::$session_id_uuid'
```

Comment:
```suggestion
    , $session_id_uuid Nullable(UInt128)
```

(See earlier comments.)